### DISCUSSION TITLE
suggestiion: implement DNSCurve proxy

### DISCUSSION THREAD

**Discussion by bjmgeek** [DNSCrypt/dnscrypt-proxy#1973] [Ideas]
According to https://dnscurve.io/faq/ the protocol for DNSCurve is pretty similar.  So, it would be nice if it was supported by this project.

**Comment by jedisct1** [comment]
dnscurve works between resolvers and authoritative servers. This is not something that runs on clients.

**Comment by bjmgeek** [comment]
Ok, thanks.
